packages/flutter/lib/src/foundation/assertions.dart

```diff
@@ -725,7 +725,17 @@ class FlutterError extends Error with DiagnosticableTreeMixin implements Asserti
   ///
   /// Set this to null to silently catch and ignore errors. This is not
   /// recommended.
-  static FlutterExceptionHandler onError = dumpErrorToConsole;
+  static FlutterExceptionHandler onError = (FlutterErrorDetails details) => presentError(details);
```

Comment:
could this just be

```suggestion
static FlutterExceptionHandler onError = presentError;
```

Reply:
This is intentional. ```static FlutterExceptionHandler onError = presentError;``` will cause an issue
when other people override `presentError`. If someone do `Flutter.presentError = some other method`,  `flutter.onError` will still use the old `presentError` instead of the new one.

Reply:
Ah, makes sense.